packages/storybook/.storybook/config.js

```diff
@@ -5,6 +5,7 @@ import { setOptions } from '@storybook/addon-options';
 setOptions({
   name: 'govuk-react',
   sortStoriesByKind: true,
+  url: 'https://ukhomeoffice.github.io/govuk-react/'
```

Comment:
IMHO it should follow storybook's example and link to the github repo, rather than the deployed storybook

Reply:
Oh yes! My mistake... will fix